pkg/server/backuprestoreserver.go

```diff
@@ -157,6 +157,8 @@ func (b *BackupRestoreServer) runServerWithSnapshotter(ctx context.Context, rest
 
 	go defragmentor.DefragDataPeriodically(ctx, b.config.EtcdConnectionConfig, b.defragmentationSchedule, ssr.TriggerFullSnapshot, b.logger)
 
+	//TODO @aaronfern: Call function for member lease renewal if enable flag is set. Needed for backward compatability
```

Comment:
I suggest to introduce the flag in this PR and enable the coding if the flag is set (disabled by default). Otherwise we introduce a lot of coding today which is not executed.